nss-cohort-37/Workforce-Green-Monkeys
# Mac training program list create

# Description

Created `TrainingProgramsController` as well as the `index` and `create` views. 

fixes #9 
## Type of change


# Testing Instructions
Run your server and navigate to the `/trainingprograms` endpoint. You should see a list of FUTURE training programs. You can click `create` and you will be given a form where you can create a new training program (note - it will only show up in your list view if it is in the future). 

Delete, detail, and edit views are not yet functional 

# Checklist:
